admin/management/views.py

```diff
@@ -119,12 +118,10 @@ def post(self, request, *args, **kwargs):
         monthly_report_date = request.POST.get('monthly_report_date', None)
         if monthly_report_date:
             report_date = isoparse(monthly_report_date).date()
-        else:
-            report_date = datetime.datetime.now().date()
 
         errors = monthly_reporters_go(
-            report_month=report_date.month,
-            report_year=report_date.year
+            report_month=getattr(report_date, 'month', None),
```

Comment:
This would throw a `NameError`  because `report_date` is defined in a conditional that won't evaluate on empty input.

Instead of having the short-lived `monthly_report_date` var, could do something like
```
report_date = request.POST.get('monthly_report_date', None)
if report_date:
    report_date = isoparse(monthly_report_date).date()
```
So that it has both a default and a parsed assignment.